libraries-bom/pom.xml

```diff
@@ -224,7 +221,7 @@
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>google-cloud-bom</artifactId>
-        <version>0.174.0</version><!-- {x-version-update:google-cloud-bom:current} -->
+        <version>0.175.0</version><!-- {x-version-update:google-cloud-bom:current} -->
```

Comment:
cc @suztomo This is close -- the PR body is not quite correct right now

Reply:
Thank you!